helm-charts/helm2/strimzi-kafka-operator/templates/_kafka_image_map.tpl

```diff
@@ -6,36 +6,41 @@
             - name: STRIMZI_DEFAULT_CRUISE_CONTROL_IMAGE
-              value: {{ default .Values.cruiseControl.image.registry .Values.imageRegistryOverride }}/{{ default .Values.cruiseControl.image.repository .Values.imageRepositoryOverride }}/{{ .Values.cruiseControl.image.name }}:{{ default .Values.cruiseControl.image.tagPrefix .Values.imageTagOverride }}-kafka-2.6.0
+              value: {{ default .Values.cruiseControl.image.registry .Values.imageRegistryOverride }}/{{ default .Values.cruiseControl.image.repository .Values.imageRepositoryOverride }}/{{ .Values.cruiseControl.image.name }}:{{ default .Values.cruiseControl.image.tagPrefix .Values.imageTagOverride }}-kafka-2.7.0
             - name: STRIMZI_DEFAULT_TLS_SIDECAR_CRUISE_CONTROL_IMAGE
-              value: {{ default .Values.tlsSidecarCruiseControl.image.registry .Values.imageRegistryOverride }}/{{ default .Values.tlsSidecarCruiseControl.image.repository .Values.imageRepositoryOverride }}/{{ .Values.tlsSidecarCruiseControl.image.name }}:{{ default .Values.tlsSidecarCruiseControl.image.tagPrefix .Values.imageTagOverride }}-kafka-2.6.0
+              value: {{ default .Values.tlsSidecarCruiseControl.image.registry .Values.imageRegistryOverride }}/{{ default .Values.tlsSidecarCruiseControl.image.repository .Values.imageRepositoryOverride }}/{{ .Values.tlsSidecarCruiseControl.image.name }}:{{ default .Values.tlsSidecarCruiseControl.image.tagPrefix .Values.imageTagOverride }}-kafka-2.7.0
             - name: STRIMZI_KAFKA_IMAGES
               value: |                 
                 2.5.0={{ default .Values.kafka.image.registry .Values.imageRegistryOverride }}/{{ default .Values.kafka.image.repository .Values.imageRepositoryOverride }}/{{ .Values.kafka.image.name }}:{{ default .Values.kafka.image.tagPrefix .Values.imageTagOverride }}-kafka-2.5.0
                 2.5.1={{ default .Values.kafka.image.registry .Values.imageRegistryOverride }}/{{ default .Values.kafka.image.repository .Values.imageRepositoryOverride }}/{{ .Values.kafka.image.name }}:{{ default .Values.kafka.image.tagPrefix .Values.imageTagOverride }}-kafka-2.5.1
                 2.6.0={{ default .Values.kafka.image.registry .Values.imageRegistryOverride }}/{{ default .Values.kafka.image.repository .Values.imageRepositoryOverride }}/{{ .Values.kafka.image.name }}:{{ default .Values.kafka.image.tagPrefix .Values.imageTagOverride }}-kafka-2.6.0
+                2.7.0={{ default .Values.kafka.image.registry .Values.imageRegistryOverride }}/{{ default .Values.kafka.image.repository .Values.imageRepositoryOverride }}/{{ .Values.kafka.image.name }}:{{ default .Values.kafka.image.tagPrefix .Values.imageTagOverride }}-kafka-2.7.0
```

Comment:
shouldn't we delete above 2.5.x? The same applies to the other env vars.

Reply:
2.5.0 is still supported. I do not see any reason for removing it since so far there are no features implemented which would not support 2.5.0 / 2.5.1.

Reply:
it confuses me because in the upgrade doc we have a lower Kafka version as 2.6 so people coming from 2.5.x could be confused as well?

Reply:
If the upgrade docs talk about upgrade from 2.5 then the users upgrading from 2.6 will be confused. I think using the latest supported Kafka from previous version makes more sense in this context.

Reply:
Anyway it's going to be confusing :)

Reply:
It will ... but I think that keeping 2.5 is still more user-friendly then removing it to he the upgrade docs fit in 100% of cases. Thanks for approving.